### DISCUSSION TITLE
How can I send data from page to page?

### DISCUSSION THREAD

**Discussion by Grumpypjs08** [Budibase/budibase#1337] [Help]
I am querying a database then want to have the option to edit I have the page but do not know how to get information about what to edit

**Comment by joebudi** [comment]
Hey @Grumpypjs08 

I am not sure what you mean. The video below provides more info re querying:
https://www.youtube.com/watch?v=Nei220VBrBU
and the video below provides more info around using parameters for querying:
https://www.youtube.com/watch?v=4-GsiKzRFk8
